plugin/package.json

```diff
@@ -9,24 +9,24 @@
   "repository": "https://github.com/videojs/generator-videojs-plugin",
   "license": "ISC",
   "dependencies": {
-    "@babel/runtime": "^7.11.2",
+    "@babel/runtime": "^7.14.0",
     "global": "^4.4.0",
-    "video.js": "^6 || ^7"
+    "video.js": "^7.12.1"
```

Comment:
I think this should stay the same

Reply:
I'm not sure that video.js 6 with support node 14

Reply:
I don't see that as an issue. The node support in Video.js is mostly in running the build itself. You definitely should be able to require vjs 6 in node 14 (and have it be a no-op, as it is now).
Ideally, this would stay the same until someone updates their plugin to not support IE11 and depend on Video.js 8.

Reply:
Reverted.